app/src/interfaces/_system/system-input-password/input-password.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+withDefaults(
+	defineProps<{
+		value?: string | null;
+		autofocus?: boolean;
+		autocomplete?: string | null;
+		disabled?: boolean;
+	}>(),
+	{
+		value: null,
+		autofocus: false,
+		autocomplete: 'new-password',
+		disabled: false,
+	},
+);
+
```

Comment:
The interface needs to define it's emits. For an interface that is the `input` event.